tests/cypress/integration/IBF-dashboard/VideoGuide.js

```diff
@@ -0,0 +1,22 @@
+import selectors from "../../support/selectors";
+import constants from "../../support/constants";
+
+// IBF Video Guide
+describe("IBF Video Guide", () => {

```

Comment:
I get an error when running this test.

<img width="397" alt="Screenshot 2021-07-26 at 11 00 53" src="https://user-images.githubusercontent.com/3880591/126963085-2912509c-76cb-41b9-b18b-e86bfb78f381.png">

Here is the stack trace,

```
    at check (webpack:///node_modules/cypress-wait-until/src/index.js:62:1)
    at resolveValue (webpack:///node_modules/cypress-wait-until/src/index.js:77:1)
    at Context.eval (webpack:///node_modules/cypress-wait-until/src/index.js:66:1)
From previous event:
    at Context.thenFn (https://ibf-test.510.global/__cypress/runner/cypress_runner.js:156453:24)
    at Context.then (https://ibf-test.510.global/__cypress/runner/cypress_runner.js:156892:22)
    at Context.<anonymous> (https://ibf-test.510.global/__cypress/runner/cypress_runner.js:171546:22)
    at <unknown> (https://ibf-test.510.global/__cypress/runner/cypress_runner.js:170970:16)
From previous event:
    at runCommand (https://ibf-test.510.global/__cypress/runner/cypress_runner.js:170949:9)
    at next (https://ibf-test.510.global/__cypress/runner/cypress_runner.js:171095:15)
From previous event:
    at next (https://ibf-test.510.global/__cypress/runner/cypress_runner.js:171095:35)
    at <unknown> (https://ibf-test.510.global/__cypress/runner/cypress_runner.js:171123:17)
```

Reply:
in commands.js we have wait request logic (cy.wait("@getHttp"); to wait for request to get completed but its getting failed  and to overcome added a line of code please check commands.js file.